assets/javascripts/discourse/components/modal/assign-user.js

```diff
@@ -0,0 +1,37 @@
+import Component from "@glimmer/component";
+import { action } from "@ember/object";
+import { inject as service } from "@ember/service";
+import I18n from "I18n";
+
+export default class AssignUser extends Component {
+  @service taskActions;
+
+  formApi = {
+    sumbit() {},
```

Comment:
```suggestion
    submit() {},
```

Reply:
me, reading/watching absolutely anything: oh, here's a typo
me, reading this suggestion: why are these lines identical

🤣